packages/datastore-storage-adapter/__tests__/ExpoSQLiteDatabase.test.ts

```diff
@@ -0,0 +1,287 @@
+import ExpoSQLiteDatabase from '../src/ExpoSQLiteAdapter/ExpoSQLiteDatabase';
+import { openDatabase } from 'expo-sqlite';
+import { ParameterizedStatement } from '../src/common/types';
+import { HelperStack } from 'typedoc/dist/lib/output/utils/resources/helpers';
+
+// mock out the underlying implementation and only keep openDatabase().
+jest.mock('expo-sqlite', () => {
+	return {
+		__esModule: true,
+		openDatabase: jest.fn(() => mockDatabase),
+	};
+});
+
+// mock out the underlying implementation
+jest.mock('expo-file-system', () => 'File System');
```

Comment:
Should be able to simplify and remove the comment (it should be obvious that this is mocking out the dependencies):
```suggestion
jest.mock('expo-sqlite', () => ({
	openDatabase: jest.fn(() => mockDatabase),
}));
jest.mock('expo-file-system');
```